Buildings/Fluid/AirFilters/BaseClasses/MassAccumulation.mo

```diff
@@ -1,13 +1,17 @@
 within Buildings.Fluid.AirFilters.BaseClasses;
 model MassAccumulation
   "Component that mimics the accumulation of the contaminants"
-  parameter Real mCon_nominal
-    "Maximum mass of the contaminant captured by the filter";
+  parameter Integer nin(
+    min=1)=1
+    "Number of input connections";
+  parameter Buildings.Fluid.AirFilters.BaseClasses.Data.Generic per
```

Comment:
I think there is no need to have the `per` parameter. We just need the nominal contaminant mass. Why don't we keep the `mCon_nominal`?

Reply:
Change it as suggested